emuu-client/src/components/home/Home.js

```diff
@@ -21,69 +20,96 @@ const theme = createTheme({
-                           <iframe width="382" height="215"
-                             src="https://www.youtube.com/embed/RrrleE-EREI"
-                             title="YouTube video player" frameborder="0"
-                           allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
-                              allowfullscreen></iframe>
-                            </div>
+        <p class="text-start">
+          <div className="break">
+            <h2>
+              <div class="p-4">Newest</div>
+            </h2>
+          </div>
 
-                           <div className = "spacer">
-                      <iframe width="382" height="215"
-                        src="https://www.youtube.com/embed/mybpNuyP9Xw"
-                         title="YouTube video player" frameborder="0"
-                        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
-                         allowfullscreen></iframe>
-                     </div>
-                    <div className = "spacer">
-                     <iframe width="382" height="215"
-                       src="https://www.youtube.com/embed/4XmfNkB8HUY"
-                        title="YouTube video player" frameborder="0"
-                      allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
-                       allowfullscreen></iframe>
-                     </div>
-                     </p>
+          <div className="spacer">
+            <iframe
+              width="382"
+              height="215"
+              src="https://www.youtube.com/embed/RrrleE-EREI"
+              title="YouTube video player"
+              frameborder="0"
+              allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+              allowfullscreen
+            ></iframe>
+          </div>
 
-            </div>
-        </ThemeProvider>
-    );
+          <div className="spacer">
+            <iframe
+              width="382"
+              height="215"
+              src="https://www.youtube.com/embed/mybpNuyP9Xw"
+              title="YouTube video player"
+              frameborder="0"
+              allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+              allowfullscreen
+            ></iframe>
+          </div>
+          <div className="spacer">
+            <iframe
+              width="382"
+              height="215"
+              src="https://www.youtube.com/embed/4XmfNkB8HUY"
+              title="YouTube video player"
+              frameborder="0"
+              allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+              allowfullscreen
+            ></iframe>
+          </div>
+        </p>
+      </div>
+    </ThemeProvider>
+  );
   useEffect = () => {
```

Comment:
Again, good job on using prettier. It really makes the code easier to read, especially for home page.